database/migrations/2020_02_21_230554_add_experience_work_table.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddExperienceWorkTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('experience_work', function (Blueprint $table) {
+            $table->bigIncrements('id');
+            $table->string('title');
+            $table->string('organization');
+            $table->string('group')->nullable();
+            $table->boolean('is_active')->nullable();
+            $table->date('start_date');
+            $table->date('end_date');
```

Comment:
I know the wireframes specify the end_date is required, but that doesn't make sense if is_active is true. I think we should double check with Gray/designers about making this nullable.